src/app/entities/courses/components/course-form/course-form.component.html

```diff
@@ -7,25 +7,57 @@ <h2 class="card__title">{{ isEditingMode ? 'Edit' : 'Create' }} course</h2>
       <label class="form-group__label">Title:</label>
       <span class="form-group__hint hint--required">required</span>
     </div>
-    <input type="text" placeholder="Text input" name="title" class="form-group__control" [(ngModel)]="courseData.name">
+    <input
+      type="text"
+      placeholder="Text input"
+      class="form-group__control"
+      [class.control--invalid]="formIsSubmitted && courseForm.get('name').invalid"
```

Comment:
looks like `courseForm.get('name')` might be a helper `getter` within this component to DRY.

Reply:
Agree, might be as an option